cmd/pfcpctl/main.go

```diff
@@ -0,0 +1,155 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022 Open Networking Foundation
+
+package main
+
+import (
+	"context"
+	"os"
+
+	pb "github.com/omec-project/pfcpsim/api"
+	"github.com/pborman/getopt/v2"
+	log "github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+const (
+	defaultAddress           = ":54321"
```

Comment:
what `:54321` is?

Reply:
Improved naming. it's the address of `pfcpsim`'s gRPC server